ansible/tasks/logging/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: win
+  tasks:
+  #- name: Logging | Install fluentd
+  #  include_tasks: install_fluentd.yml
+  #
+  - name: Logging | Install fluentd plugins
+    include_tasks: install_plugins.yml
+
+  #- name: Logging | Configure
+  #  include_tasks: configure.yml
```

Comment:
What's with the commented out code?

Reply:
It was meant to be a guideline as to how this piece of code fits in to the greater playbook. I can either add TODOs, or remove the commented out code if needed